examples/data-objects/prosemirror/src/fluidBridge.ts

```diff
@@ -238,7 +238,7 @@ export class ProseMirrorTransactionBuilder {
 
         for (const thing of this.things) {
             if (thing.type === "ether") {
-                if (thing.annotations) {
+                if (thing.annotations !== undefined) {
```

Comment:
These might be fine, but I'm always nervous about integration with sequence/mergetree because they use `null` more than you'd expect.  Is there any risk here (and below) are `null` rather than `undefined`?

Reply:
Yeah, this is a great point. I think this is one of the biggest flaws with strict-boolean-expressions -- coercion neatly handles some of these cases. Not arguing for turning the rule off but I've been struggling to apply the correct fix for each case, which to me means it's more likely other folks will make similar mistakes when trying to fix the lint error.